video/out/vo_gpu_next.c

```diff
@@ -963,7 +964,7 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
         struct pl_queue_params qparams = *pl_queue_params(
             .pts = frame->current->pts + pts_offset,
             .radius = pl_frame_mix_radius(&params),
-            .vsync_duration = frame->ideal_frame_vsync_duration,
+            .vsync_duration = can_interpolate ? frame->ideal_frame_vsync_duration : 0,
```

Comment:
Why does `vo.c` even set wrong values in the `display_synced` case? Wouldn't it make more sense to make vo.c sanitize this struct when no DS is requested?

Reply:
So wrong is strong word, because I believe it would actually be zeroed. But the value make sense only if interpolating, because it is duration of frames divided by number of vsyncs, when not interpolating we don't want to use this value.

And yes I agree `can_interpolate` can be removed and in general more improvement can be made. But my goal is to merge this PR before release, which apparently is imminent. Hence I opted for minimal changes to make it work and contained changes only to gpu_next.

Don't worry I will bikeshed (not a word) it more after release.  I know what are the pain points now, it is just that for release I don't want change vo.c